app/components/recirculation-prefooter.js

```diff
@@ -56,6 +88,17 @@ export default Component.extend(
 							'LI'
 						);
 					});
+
+					if (this.get('shouldShowPlista')) {
+						this.fetchPlista()
+							.then(this.mapPlista)
+							.then((item) => {
+								if (item.thumbnail) {
+									this.notifyPropertyChange('items');
+									this.set('items.1', item);
```

Comment:
If Plista is slow to return the data, it's possible we display a Fandom article and then replace it with an ad, right? IMO it's a poor UX, user might like the article, wanted to click it and it disappeared. It would be better to inject ad between the existing items.